hotspot/test/runtime/containers/cgroup/CgroupSubsystemFactory.java

```diff
@@ -158,6 +163,12 @@ private void setup() {
 
             cgroupV2MntInfoMissingCgroupv2 = Paths.get(existingDirectory.toString(), "mnt_info_missing_cgroup2");
             Files.writeString(cgroupV2MntInfoMissingCgroupv2, mntInfoHybridStub);
+
+            cgroupv1MntInfoDoubleCpuset = Paths.get(existingDirectory.toString(), "mnt_info_cgroupv1_double_cpuset");
+            Files.writeString(cgroupv1MntInfoDoubleCpuset, mntInfoCgroupv1DoubleCpuset);
```

Comment:
Without actually testing the patch. This looks like API not available in 8u. `s/Files.writeString/Files.write/`. Once the depending PR gets updated this would likely need some fix-up as well. I'll looking at recent changes from #135.